yellowbrick/utils.py

```diff
@@ -155,6 +157,45 @@ def is_dataframe(obj):
 isdataframe = is_dataframe
 
 
+#From here: http://stackoverflow.com/questions/26248654/numpy-return-0-with-divide-by-zero
+def numpy_div0( a, b ):
+    """
+    Ufunc-extension that returns 0 instead of nan when dividing numpy arrays
+    example: >> div0( [-1, 0, 1], 0 ) -> [0, 0, 0]
+    """
+    with np.errstate(divide='ignore', invalid='ignore'):
+        c = np.true_divide( a, b )
+        c[ ~ np.isfinite( c )] = 0  # -inf inf NaN
+    return c
+
+
+def find_text_color(base_color, text_colors=("black","white")):
+    '''
+    Takes a background color and returns the appropriate light or dark text color.
+    Users can specify the dark and light text color, or accept the defaults of 'black' and 'white'
+
+    # baseColor: the color of the background, in any valid color specification format
+    # textColors: a tuple of (dark, light) text format. 
+    '''
+
+    #matplotlib cmap exports rgba colors scaled 0-1 instead of standard 255
+    rgb=[]
+    for idx,val in enumerate(base_color):
+        rgb.append(val*255)
+    
+    #Calculate brightness relative to the color type
+    base_brightness = math.sqrt( .241*rgb[0]*rgb[0]+
```

Comment:
So perhaps you mentioned this in the related issue, but I'm not sure where the coefficients used in this luminescence algorithm  come from. For example, I found the following: `sqrt(0.299 * R^2 + 0.587 * G^2 + 0.114 * B^2)` from http://stackoverflow.com/questions/596216/formula-to-determine-brightness-of-rgb-color -- but there are clearly other options, including for gamma-correction, etc. 

One way to deal with this would be to put the coefs in a variable, then we could change them with different arguments in the function signature:

```python
coefs = np.array((.241, .691, .068))
rgb = np.array(rgb)
brightness = np.sqrt(np.dot(coefs, rgb**2))
```